### PR TITLE
Warning banner

### PR DESCRIPTION
**Pull request by duggiemitchell**
see: https://trello.com/c/6DTSL5Pt

Image of Yaktocat

### CODE REVIEW

File: src/js/components/infoToast.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    const { link } = this.props;
+    const { isOpen } = this.state;
+    return (
+      <>
+        {isOpen ? (
+          <ToastContainer>
+            <Toast
+              labels={{
+                heading: [
+                  'This is not the most recent version of this product. ',
```
Comment (wlonk) [inline src/js/components/infoToast.js]: Does this need to be wrapped in translation magic?

File: src/js/components/infoToast.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                  <Link key="version-link " to={link}>
+                    Switch to the most recent version.
```
Comment (wlonk) [inline src/js/components/infoToast.js]: Ditto this.

**Comment by mirisuzanne** [comment]
[Show warning banner if on Version or Plan for a version other than the latest version of the Product. Include a link to latest Version or Plan.](https://trello.com/c/6DTSL5Pt/151-show-warning-banner-if-on-version-or-plan-for-a-version-other-than-the-latest-version-of-the-product-include-a-link-to-latest-ve)

**Comment by jgerigmeyer** [comment]
@wlonk This is ready for final code review.

**Pull request merged**